### PR TITLE
Archive gifs

### PR DESCRIPTION
**Pull request by oneroyalace**
This PR allows the archiving of GIFs from Twitter, which are processed/archived just like videos are (using Twitter's 1.1 API for now). The PR also adds a new attribute to `Birdsong::Tweet`, `video_file_type` to distinguish between Tweets that contain gifs and Tweets that contain normal video types (e.g., mp4) 

### Testing
1. `bundle install`
2. `rake test`

### CODE REVIEW

**Review by cguess — APPROVED** [review]
Looks great with just a few questions and a bye bug that needs to get removed. Consider approved otherwise

File: .rubocop.yml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 require:
-  - rubocop-rails
   - rubocop-performance
```
[the remark this answers is not in the document]
Comment (oneroyalace) [reply .rubocop.yml:2]: Oops, this was a mistake. Restored

File: lib/birdsong/tweet.rb [hunk 2]
```diff
@@ -64,26 +65,31 @@ def parse(json_tweet, includes)
       end.compact # compact because of the `next` above will return `nil`
 
       @video_file_names = media_items.map do |media_item|
-        next unless media_item["type"] == "video"
-
+        next unless (media_item["type"] == "video") || (media_item["type"] == "animated_gif")
```
[the remark this answers is not in the document]
Comment (oneroyalace) [reply lib/birdsong/tweet.rb:68]: I don't. Operator precedence should carry us through. I'll change that.

File: test/tweet_test.rb [hunk 3]
```diff
@@ -69,4 +70,13 @@ def test_that_a_tweet_handles_no_variants_for_video
     assert_equal 1, tweet.video_file_names.count
     assert_equal 0, tweet.image_file_names.count
   end
+
+  def test_that_a_tweet_can_have_a_gif
+    tweet = Birdsong::Tweet.lookup("1472873480249131012").first
+    byebug
```
[the remark this answers is not in the document]
Comment (oneroyalace) [reply test/tweet_test.rb:76]: 😬 removed

**Pull request merged**